### PR TITLE
Issue 7683: Update JPA feature descriptions

### PR DESCRIPTION
**Pull request by dazey3**
for #7683 

Signed-off-by: Will Dazey <dazeydev.3@gmail.com>

### CODE REVIEW

**Comment by dazey3** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/jpa-2.1/resources/l10n/com.ibm.websphere.appserver.jpa-2.1.properties [hunk 1]
```diff
@@ -14,6 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables support for applications that use application-managed and container\
--managed JPA written to the Java Persistence API 2.1 specification. The support is built on top of \
- EclipseLink to support the container-managed programming model.
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This includes Java Persistence API 2.1 specification interfaces, \
+container-managed JPA integration, and EclipseLink as the default JPA provider implementation.
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jpa-2.1/resources/l10n/com.ibm.websphere.appserver.jpa-2.1.properties]: **change**
`This includes Java Persistence API 2.1 specification interfaces, \
container-managed JPA integration, and EclipseLink as the default JPA provider implementation.`
**to**
`This support includes Java Persistence API 2.1 specification interfaces and 
container-managed JPA integration. EclipseLink is also supported, as the default JPA provider implementation.`

File: dev/com.ibm.websphere.appserver.features/visibility/public/jpa-2.2/resources/l10n/com.ibm.websphere.appserver.jpa-2.2.properties [hunk 2]
```diff
@@ -14,6 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables support for applications that use application-managed and container\
--managed JPA written to the Java Persistence API 2.2 specification. The support is built on top of \
- EclipseLink to support the container-managed programming model.
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This includes Java Persistence API 2.2 specification interfaces, \
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jpa-2.2/resources/l10n/com.ibm.websphere.appserver.jpa-2.2.properties]: **change**
`This includes Java Persistence API 2.2 specification interfaces, container-managed JPA integration, and EclipseLink as the default JPA provider implementation.`
 
**to**
`This support includes Java Persistence API 2.2 specification interfaces and 
container-managed JPA integration. EclipseLink is also supported, as the default JPA provider implementation.`

File: dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-2.1/resources/l10n/com.ibm.websphere.appserver.jpaContainer-2.1.properties [hunk 3]
```diff
@@ -14,7 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables support for applications that use application-managed\
-and container-managed JPA written to the Java Persistence API 2.1 specification.  This \
-only includes Java Persistence API 2.1 specification interfaces, and container-managed\
-JPA integration.  This feature does not include any JPA implementation.
\ No newline at end of file
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This only includes Java Persistence API 2.1 specification interfaces \
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-2.1/resources/l10n/com.ibm.websphere.appserver.jpaContainer-2.1.properties]: **change**
`This only includes Java Persistence API 2.1 specification interfaces and container-managed JPA integration.`
**to**
`This support includes only Java Persistence API 2.1 specification interfaces and container-managed JPA integration.`

File: dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-2.2/resources/l10n/com.ibm.websphere.appserver.jpaContainer-2.2.properties [hunk 4]
```diff
@@ -14,7 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables support for applications that use application-managed\
-and container-managed JPA written to the Java Persistence API 2.2 specification.  This \
-only includes Java Persistence API 2.2 specification interfaces, and container-managed\
-JPA integration.  This feature does not include any JPA implementation.
\ No newline at end of file
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This only includes Java Persistence API 2.2 specification interfaces \
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-2.2/resources/l10n/com.ibm.websphere.appserver.jpaContainer-2.2.properties]: **change**
`This only includes Java Persistence API 2.2 specification interfaces and container-managed JPA integration.`
**to**
`This support includes only Java Persistence API 2.2 specification interfaces and container-managed JPA integration.`

**Comment by dazey3** [comment]
Thanks for the review @dmuelle !

**Pull request merged**